components/atom/backToTop/src/utils.js

```diff
@@ -0,0 +1,34 @@
+const isReactRefObj = target => {
```

Comment:
If you gonna reuse this, it's ok to move this to a `react-components` https://github.com/SUI-Components/sui/tree/master/packages/sui-js/src folder or something like that

Reply:
https://github.com/SUI-Components/sui/pull/411